convert/helpers.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import open3d as o3d
+
+def get_point_cloud(input_path, dataname):
+    # Load the point cloud from the .pcd file
+    point_cloud = o3d.io.read_point_cloud(input_path+dataname)
+
+    # Extract points as a NumPy array
+    points = np.asarray(point_cloud.points)
+
+    # If needed, perform slicing or other operations on the NumPy array
+    points_subset = points[:, :3] 
+
+    # Convert to Open3D PointCloud
+    pcd = o3d.geometry.PointCloud()
+    pcd.points = o3d.utility.Vector3dVector(points_subset)
+
+    # Estimate normals for pcd directly
+    pcd.estimate_normals(search_param=o3d.geometry.KDTreeSearchParamHybrid(radius=0.1, max_nn=30))
```

Comment:
Can we please put these in command line args?

Reply:
I can certainly add this.
Are you also thinking that these values should be specified via attributes?